helpdesk_mgmt_sale/views/sale_order_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 Tecnativa - Pilar Vargas
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='sale_reporting']/.." position="after">
+                <group>
```

Comment:
Don't create a new super `group` instead of inserting it inside another existing one. You can do `group[@name='...']/..` to get the parent group of for example the "Sales" one.

Reply:
OK, I'd better put it behind ‘Tracking’ without creating a super group, otherwise the delivery date gets stuck in the middle and it looks a bit weird.
This is how it would look:
![image](https://github.com/user-attachments/assets/f0ed4f26-713d-40f2-8af3-6dd217bb410c)

Reply:
"Delivery date" seems a bit weird. Maybe you can put it after "Invoicing" group.